### ISSUE TITLE
Uncaught SyntaxError: Unexpected token <

### ISSUE THREAD

**Issue by kuya1284** [kaltura/platform-install-packages#403]
When trying to view the KMC, all I'm seeing is the top navigation (with all the tabs), but the main content area is just a blank white page. When using my (Chrome) browser's developer tools, the following error message appears:

> Uncaught SyntaxError: Unexpected token <

While looking at the source, it appears that PHP short open tags are used in various parts of the code. (i.e. *<?* instead of *<?php*).

In order to get past this, I went ahead and enabled *short_open_tag* in my php.ini, the entire page breaks as if all static content didn't get loaded (i.e. CSS, JS, etc.).

I tested this in Ubuntu 12.04, 14.04, and CentOS 7. Could you please advise me on what might be wrong? Thanks.

**Comment by DBezemer** [comment]
This typically means that the installation was not successful, or that you have not executed kaltura-config-all.sh, so please check the installation logs, and/or run the config-all.sh script again

**Comment by kuya1284** [comment]
Thanks for the response. I've actually done just that many times, and followed the guide exactly as instructed, but still can't get the KMC to load correctly. I've spent days on this and have never had this much trouble installing an application.  

In CentOS, when I rerun /opt/kaltura/bin/kaltura-config-all.sh, I also noticed that some of the prompts/questions that appear during a clean install don't appear after subsequently running that install script again.  Also, there are other prompts/questions that do appear that weren't even asked the first time around.  Deleting base-config.lock essentially had no effect on this.  I even went as far as running POPULATE_ONLY=1 /opt/kaltura/bin/kaltura-db-config.sh, but that seems to stop when insertingContent. The insertContent.log file doesn't appear to contain anything useful other than:

> PHP Fatal error:  Uncaught exception 'KalturaClientException' with message 'failed to unserialize server result

In Ubuntu 14.04, everything install fine. The only issue that I was having was not being able to install Red5. No errors or anything. However, when trying to access the KMC, I still get that blank white page in the content area. The tabs are present as previously mentioned. No matter how I slice and dice this, the KMC won't load.  

I'm not sure what else to do. I wasn't hoping for a better experience with CentOS, but it was similar to Ubuntu. I'd prefer to stick with Ubuntu at this point since that's what I'm more comfortable with in terms of the CLI.

**Comment by kuya1284** [comment]
Oh, and another thing I noticed in Ubuntu is that the system keeps trying to send an email to **kaltura@[mydomain].org**.  I never entered this during the setup.  An error message keeps bouncing back to me indicating that delivery to this email address was unsuccessful, and rightfully so.

**Comment by jessp01** [comment]
Can you please run kaltlog on the server end while making the request and paste the errors you are seeing? Also, in parallel, run an HTTP sniffer to see what calls are put out and which one generates the HTTP error 500.

RE kaltura@[mydomain].org, I believe this might be a cron job running as the kaltura user and failing. Check /var/log/cron or /var/log/syslog to see what job it is.

**Comment by jessp01** [comment]
As for short tags, nothing in the code should use them, if you can point me to the code that does, we can fix it but on my Ubuntu box
# php -i|grep short
short_open_tag => Off => Off

And everything works,

**Comment by kuya1284** [comment]
Hi Jess,

The files that contains the PHP short open tags are:

* /opt/kaltura/app/alpha/apps/kaltura/modules/system/templates/gallerySuccess.php (line 190)
* /opt/kaltura/app/alpha/apps/kaltura/modules/system/templates/galleryPartialSuccess.php (line 69)
* /opt/kaltura/app/alpha/apps/kaltura/modules/kmc/templates/kmc1Success.php (lines 217, 233, 247, 328, 329, 349, 350, 351, 361, 363, 373, 374, 391, 392, 393, 394, 404, 406, 416, 417, 441, 443, 454, 455, 472, 474, 484, 486, 497, 498, 509, 521, 568, 569, 591, 594, 642, 644, 645, 646, 647, 648, 671, 779, 854, 901, 927, 936, 945, and 971)

Also, there's another issue with kmc1Success.php. On lines 927, 936, and 945, if $player_uiconfg_list is null, or if the array at index 0 is empty, these lines fail.  If $player_uiconf_list[0] is not an object, PHP will throw an error. Thus, these lines should probably be updated to check that the object has been set.

I'll take a look at the logs more and see what I can dig up. Thanks.

**Comment by KalturaCommunity** [comment]
Great, since you are the one who found it, can you please submit a pull 
request so the credit can properly go to you? Thank you,

May the source be with you,

Jess Portnoy

Join us for the Kaltura Connect Virtual Summits 2015 - Register now!
connect.kaltura.com

**Comment by kuya1284** [comment]
Before doing any sniffing, I thought I'd just check kaltura_apache_errors.log first. I noticed the following:

> AH01630: client denied by server configuration: /opt/kaltura/app/kmc

When I go into /opt/kaltura/app, I don't see a **kmc** directory. Is there supposed to be one in there or is there a rewrite rule that points http://[mydomain]/kmc to a different route/path?

**Comment by jessp01** [comment]
Its rewrite rules. You can see it under /opt/kaltura/app/configurations/apache

Regardless, none of our code should have shorttags. Thanks,


May the source be with you,

Jess Portnoy



-------- Original message --------

**Comment by kuya1284** [comment]
Jess, I don't know if this is the problem, but in the apache config apps.conf, I see the following alias:

> Alias /kmc "/opt/kaltura/app/kmc/web"

However, **/opt/kaltura/app/kmc** doesn't exist.  I also noticed that after correcting all the short tag issues, I'm now getting a 404 for the Dashboard:

> GET http://[mydomain]/flash/kmc/dashboard/v1.0.1/dashboard.swf 404 (Not Found)

**Comment by kuya1284** [comment]
I also tried searching for dashboard.swf in /opt/kaltura/web/flash/, but nothing came up.

**Comment by jessp01** [comment]
Can you please run kaltlog and also a sniffer and see what's going on?

**Comment by jessp01** [comment]
Hello @kuya1284,
Any update? thanks,

**Comment by kuya1284** [comment]
Hi Jess,

My only update was what I posted in regards to the missing **dashboard.swf** file.  I can't spend anymore time than I already have to be honest.  I was just trying to setup the app to give it a test run to see if it would suit our needs. I just don't understand why I have to jump through so many hoops just to get something basic up and running.  I'm willing to give this another shot once all these kinks have been ironed out, but for no, I just don't have the time to spend on troubleshooting unfortunately.

Thanks.

**Comment by jessp01** [comment]
Closing but please reopen should you be able to invest time collecting more data.

**Comment by damir-oiv** [comment]
I had also haad this problem with <?php and <? which I resolved via apache setting and now I have the problem with browser unable to load swf files, e.g. dashboard.swf file

For example KMC page source is having this link (x.y.z is my server): 
http://x.y.z.hr/flash/kmc/dashboard/v1.0.1/dashboard.swf

When i do: find /opt/kaltura/ | grep -i dash | grep -i swf

I get:
/opt/kaltura/web/flash/kmc/v5.40.1/locale/ja_JP/dashboard_ja_JP_ResourceModule.swf
/opt/kaltura/web/flash/kmc/v5.40.1/locale/keys/dashboard_keys_ResourceModule.swf
/opt/kaltura/web/flash/kmc/v5.40.1/locale/es_ES/dashboard_es_ES_ResourceModule.swf
/opt/kaltura/web/flash/kmc/v5.40.1/locale/fr_FR/dashboard_fr_FR_ResourceModule.swf
/opt/kaltura/web/flash/kmc/v5.40.1/locale/en_US/dashboard_en_US_ResourceModule.swf
/opt/kaltura/web/flash/kmc/v5.40.1/locale/de_DE/dashboard_de_DE_ResourceModule.swf
/opt/kaltura/web/flash/kmc/v5.40.1/modules/Dashboard.swf

Can you please help?

**Comment by damir-oiv** [comment]
tail -f /opt/kaltura/log/kaltura_apache_errors.log:

[Thu Mar 10 15:36:00.732860 2016] [autoindex:error] [pid 24431] [client 82.193.207.68:14460] AH01276: Cannot serve directory /opt/kaltura/web/flash/kmc/: No matching DirectoryIndex (index.html) found, and server-generated directory index forbidden by Options directive
[Thu Mar 10 15:36:08.060099 2016] [authz_core:error] [pid 24433] [client 82.193.207.68:14534] AH01630: client denied by server configuration: /opt/kaltura/app/kmc, referer: http://x.y.z.hr/start/index.php
[Thu Mar 10 15:36:08.087974 2016] [authz_core:error] [pid 24433] [client 82.193.207.68:14534] AH01630: client denied by server configuration: /opt/kaltura/app/kmc, referer: http://x.y.z.hr/start/index.php
[Thu Mar 10 15:36:08.118277 2016] [:error] [pid 24433] [client 82.193.207.68:14534] PHP Notice:  Undefined offset: 0 in /opt/kaltura/app/alpha/apps/kaltura/modules/kmc/templates/kmc1Success.php on line 927, referer: http://x.y.z.hr/start/index.php
[Thu Mar 10 15:36:08.118346 2016] [:error] [pid 24433] [client 82.193.207.68:14534] PHP Fatal error:  Call to a member function getId() on null in /opt/kaltura/app/alpha/apps/kaltura/modules/kmc/templates/kmc1Success.php on line 927, referer: http://oiv-srv-kaltura02.iptv.oiv.hr/start/index.php
[Thu Mar 10 15:36:09.704121 2016] [authz_core:error] [pid 24476] [client 82.193.207.68:14560] AH01630: client denied by server configuration: /opt/kaltura/app/kmc, referer: http://x.y.z/start/index.php
[Thu Mar 10 15:36:09.734020 2016] [:error] [pid 24476] [client 82.193.207.68:14560] PHP Notice:  Undefined offset: 0 in /opt/kaltura/app/alpha/apps/kaltura/modules/kmc/templates/kmc1Success.php on line 927, referer: http://x.y.z/start/index.php
[Thu Mar 10 15:36:09.734102 2016] [:error] [pid 24476] [client 82.193.207.68:14560] PHP Fatal error:  Call to a member function getId() on null in /opt/kaltura/app/alpha/apps/kaltura/modules/kmc/templates/kmc1Success.php on line 927, referer: http://x.z.y/start/index.php

**Comment by DBezemer** [comment]
@damir-oiv this would happen if you login to KMC with the user/email you used to create the installation. Login to the Admin Console (/admin_console/) instead and create a new publisher first.

You can then login from the Admin Console to the KMC, and that way your account gets associated to the KMC account.

**Comment by damir-oiv** [comment]
You are right. Thx!

Now when I login to kmc as a publisher, I get internal server error. The apache error log says:

[Fri Mar 11 10:29:35.466386 2016] [authz_core:error] [pid 8167] [client 82.193.207.68:38745] AH01630: client denied by server configuration: /opt/kaltura/app/kmc, referer: http://x.y.z/index.php/kmc/kmc
[Fri Mar 11 10:29:35.506804 2016] [authz_core:error] [pid 8170] [client 82.193.207.68:38730] AH01630: client denied by server configuration: /opt/kaltura/app/kmc, referer: http://x.y.z/index.php/kmc/kmc
[Fri Mar 11 10:29:35.537139 2016] [authz_core:error] [pid 8170] [client 82.193.207.68:38730] AH01630: client denied by server configuration: /opt/kaltura/app/kmc, referer: http://oiv-srv-kaltura02.iptv.oiv.hr/index.php/kmc/kmc
[Fri Mar 11 10:29:35.869072 2016] [authz_core:error] [pid 20241] [client 82.193.207.68:38753] AH01630: client denied by server configuration: /opt/kaltura/app/kmc, referer: http://x.y.z/index.php/kmc/kmc4
[Fri Mar 11 10:29:35.871690 2016] [authz_core:error] [pid 19984] [client 82.193.207.68:38754] AH01630: client denied by server configuration: /opt/kaltura/app/kmc, referer: http://x.y.z/index.php/kmc/kmc4

**Comment by damir-oiv** [comment]
Also noticed some logs in kaltura api log related to reports
2016-03-11 10:39:05 [0.000488] [82.193.207.68] [863993003] [40] [API] [kCoreException->__construct] ERR: exception 'kCoreException' with message 'Invalid query' in /opt/kaltura/app/alpha/apps/kaltura/lib/reports/myReportsMgr.class.php:1330
2016-03-11 10:39:21 [0.000810] [10.23.1.60] [845460967] [14] [API] [KalturaStatement->execute]

**Comment by damir-oiv** [comment]
Also tried to run scripts (as user kaltura) from /etc/cron.d/kaltura-dwh
All scripts run fine, except the last one which reports:

2016-03-11 10:42:21 [require_once] INFO: Starting script
2016-03-11 10:42:21 [require_once] INFO: Initializing database...
2016-03-11 10:42:21 [require_once] INFO: Database initialized successfully
2016-03-11 10:42:21 [KalturaPDO->__construct] DEBUG: conn took - 0.0034229755401611 seconds to mysql:host=localhost;port=3306;dbname=kaltura;
2016-03-11 10:42:21 [Propel::initConnection] NOTICE: total conn took 0.0055949687957764 mysql:host=localhost;port=3306;dbname=kaltura;
start dwh wrap
ERROR 1305 (42000) at line 1: PROCEDURE kalturadw.mark_operational_sync_as_done does not exist
end dwh

**Comment by jessp01** [comment]
2016-03-11 10:39:05 [0.000488] [82.193.207.68] [863993003] [40] [API] [kCoreException->__construct] ERR: exception 'kCoreException' with message 'Invalid query' in /opt/kaltura/app/alpha/apps/kaltura/lib/reports/myReportsMgr.class.php:1330

Indicates that there is some issue with your DWH DBs deployment. Basically, it cannot connect to get the stats. And indeed, you have a missing procedure:
 ERROR 1305 (42000) at line 1: PROCEDURE kalturadw.mark_operational_sync_as_done does not exist

Seems like your DB deployment is incomplete. I suggest you run:
\# kaltura-drop-db,sh
and then re-run kaltura-config-all.sh to redeploy it.

Also, we prefer to keep general questions and troubleshooting at forum.kaltura.org and keep the Github issues section to general bug reporting.

I am therefore closing this issue but feel free to post additional questions at forum.kaltura.org.

Thanks,

**Comment by damir-oiv** [comment]
It says

./kaltura-config-all.sh
Running base config...



/opt/kaltura/bin/kaltura-base-config.sh: line 69: rpm: command not found


I am on Debian :)

**Comment by KalturaCommunity** [comment]
Alright:) On Debian, you just need to re-run the install script post 
removing the DB. It will already skip all the prompting because they are 
handled by debconf and will simply move on to rebuilding the DB which 
happens during the kaltura-db postinst script.

May the source be with you,

Jess Portnoy

Kaltura Connect Virtual Summits 2015 - now on VOD! [connect.mediaspace.kaltura.com]

**Comment by damir-oiv** [comment]
Hi! Did: ./install_kaltura_all_in_1.sh

It said:
No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

'non-free' distribution component is already enabled for all sources.
No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

No packages will be installed, upgraded, or removed.
0 packages upgraded, 0 newly installed, 0 to remove and 94 not upgraded.
Need to get 0 B of archives. After unpacking 0 B will be used.

[ ok ] Starting kaltura-nginx (via systemctl): kaltura-nginx.service.

**Comment by jessp01** [comment]
Did you run kaltura-drop-db.sh beforehand?

May the source be with you,

Jess Portnoy

Kaltura Connect Virtual Summits 2015 - now on VOD! [connect.mediaspace.kaltura.com]

**Comment by damir-oiv** [comment]
Yep I did!
Wanted to do it again, but:
./kaltura-drop-db.sh
This will drop the following DBs:
kaltura kaltura_sphinx_log kalturadw kalturadw_ds kalturadw_bisources kalturalog
and remove users:
kaltura etl
on localhost
NOTE: this is not reversible.
It is recommended you also back up the current data using mysqldump before continuing.
You can use /opt/kaltura/bin/kaltura-export-db.sh to export the data.

Are you absolutely certain you want this? [n/Y]


Y
root DB passwd:
ERROR 1396 (HY000) at line 1: Operation DROP USER failed for 'kaltura'@'%'

**Comment by KalturaCommunity** [comment]
Hi,

What's the output for:
mysql> show databases;
and:
mysql> select user,host from mysql.user;

Please run both commands using MySQL's super user.

May the source be with you,

Jess Portnoy

Kaltura Connect Virtual Summits 2015 - now on VOD! [connect.mediaspace.kaltura.com]

**Comment by damir-oiv** [comment]
mysql> show databases;
+--------------------+
| Database           |
+--------------------+
| information_schema |
| mysql              |
| performance_schema |
+--------------------+
3 rows in set (0.02 sec)

mysql> select user,host from mysql.user;
+------------------+-------------------+
| user             | host              |
+------------------+-------------------+
| root             | 127.0.0.1         |
| root             | ::1               |
| debian-sys-maint | localhost         |
| root             | localhost         |
| root             | xxx-srv-kaltura02 |
+------------------+-------------------+
5 rows in set (0.00 sec)

**Comment by KalturaCommunity** [comment]
Good. So the Kaltura DBs and users have been dropped.
What happens if you run:
# dpkg-reconfigure kaltura-db
?

It should detect that you do not have a DB and deploy a new one.

May the source be with you,

Jess Portnoy

Kaltura Connect Virtual Summits 2015 - now on VOD! [connect.mediaspace.kaltura.com]

**Comment by damir-oiv** [comment]
dpkg-reconfigure kaltura-db

Checking MySQL version..

CREATE USER kaltura;
CREATE USER etl@'%';
CREATE DATABASE kaltura;
CREATE DATABASE kaltura_sphinx_log;
CREATE DATABASE kalturadw;
CREATE DATABASE kalturadw_ds;
CREATE DATABASE kalturadw_bisources;
CREATE DATABASE kalturalog;
Checking connectivity to needed daemons...
Connectivity test passed:)
Cleaning cache..
Populating DB with data.. please wait..
Output for /opt/kaltura/app/deployment/base/scripts/installPlugins.php being logged into /opt/kaltura/log/installPlugins.log
Output for /opt/kaltura/app/deployment/base/scripts/insertDefaults.php being logged into /opt/kaltura/log/insertDefaults.log
Output for /opt/kaltura/app/deployment/base/scripts/insertPermissions.php being logged into /opt/kaltura/log/insertPermissions.log
Output for /opt/kaltura/app/deployment/base/scripts/insertContent.php being logged into /opt/kaltura/log/insertContent.log


kaltura-db.postinst FAILED with: 3113 on line 180

Archving logs to /opt/kaltura/log/log_11_03_16_15_24.tar.gz...

**Comment by KalturaCommunity** [comment]
Right, so it is failing.
Take a look at /opt/kaltura/log/insertContent.log
and check why.

My guess; you provided a hostname/service URL that is not resolvable and 
therefore, the DB deployment, which makes API calls to do its work, is 
failing.

May the source be with you,

Jess Portnoy

Kaltura Connect Virtual Summits 2015 - now on VOD! [connect.mediaspace.kaltura.com]

**Comment by damir-oiv** [comment]
execution time for [http://xx-srv-kaltura02.xxx.iv.hr/api_v3/service/multirequest]: [0.1117959022522]
Queuing action [userRole.add]
Queuing action [user.add]
Executing multirequest
service url: [http://xxx-srv-kaltura02.iptv.xxx.hr]
curl: http://xxx-srv-kaltura02.iptv.xxx.hr/api_v3/service/multirequest
post: {"format":"3","ignoreNull":true,"clientTag":"php5:16-03-10","apiVersion":"3.3.0","0":{"service":"userrole","action":"add","userRole":{"objectType":"KalturaUserRole","name":"System Administrator","systemName":"System Administrator","description":"System Administrator","status":"1","permissionNames":"*","tags":"admin_console"},"ks":"YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow=="},"1":{"service":"user","action":"add","user":{"objectType":"KalturaUser","id":"xxx","screenName":"xxx","fullName":"Kaltura Administrator","email":"xxx","status":"1","password":"xxx","isAdmin":"1","roleIds":"{1:result:id}","allowedPartnerIds":"*"},"ks":"YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow=="},"kalsig":"d47f061a573fde3af0e2d5d4a79b6644"}
server: [xxx-srv-kaltura02], session: [1343418808]
result (serialized): a:2:{i:0;a:4:{s:4:"code";s:10:"INVALID_KS";s:7:"message";s:129:"Invalid KS "YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow==", Error "-1,INVALID_STR"";s:10:"objectType";s:19:"KalturaAPIException";s:4:"args";a:3:{s:4:"KSID";s:92:"YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow==";s:8:"ERR_CODE";i:-1;s:8:"ERR_DESC";s:11:"INVALID_STR";}}i:1;a:4:{s:4:"code";s:10:"INVALID_KS";s:7:"message";s:129:"Invalid KS "YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow==", Error "-1,INVALID_STR"";s:10:"objectType";s:19:"KalturaAPIException";s:4:"args";a:3:{s:4:"KSID";s:92:"YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow==";s:8:"ERR_CODE";i:-1;s:8:"ERR_DESC";s:11:"INVALID_STR";}}}
result (object dump): Array
(
    [0] => Array
        (
            [code] => INVALID_KS
            [message] => Invalid KS "YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow==", Error "-1,INVALID_STR"
            [objectType] => KalturaAPIException
            [args] => Array
                (
                    [KSID] => YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow==
                    [ERR_CODE] => -1
                    [ERR_DESC] => INVALID_STR
                )

        )

    [1] => Array
        (
            [code] => INVALID_KS
            [message] => Invalid KS "YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow==", Error "-1,INVALID_STR"
            [objectType] => KalturaAPIException
            [args] => Array
                (
                    [KSID] => YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow==
                    [ERR_CODE] => -1
                    [ERR_DESC] => INVALID_STR
                )

        )

)

execution time for [http://xxx-srv-kaltura02.iptv.xxx.hr/api_v3/service/multirequest]: [0.030406951904297]
Executing failed for request #1 with error [Invalid KS "YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow==", Error "-1,INVALID_STR"]
PHP Fatal error:  Uncaught exception 'KalturaException' with message 'Invalid KS "YThhNjRlNDM5NzkyNzk4MTg0MzNmOTNjZjRkYzA2NzRhNTIwNmEwMHwtMjstMjsxNDU3NzkyMzQyOzI7MjE0MzY7Ow==", Error "-1,INVALID_STR"' in /opt/kaltura/app/tests/standAloneClient/exec.php:351
Stack trace:
#0 {main}
  thrown in /opt/kaltura/app/tests/standAloneClient/exec.php on line 351
Queuing action [userRole.add]
Queuing action [user.add]
Executing multirequest
service url: [http://xxx-srv-kaltura02.iptv.oivxxxhr]
curl: http://xxx-srv-kaltura02.iptv.xxx.hr/api_v3/service/multirequest
post: {"format":"3","ignoreNull":true,"clientTag":"php5:16-03-10","apiVersion":"3.3.0","0":{"service":"userrole","action":"add","userRole":{"objectType":"KalturaUserRole","name":"System Administrator","systemName":"System Administrator","description":"System Administrator","status":"1","permissionNames":"*","tags":"admin_console"},"ks":"ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow=="},"1":{"service":"user","action":"add","user":{"objectType":"KalturaUser","id":"xxx","screenName":"xxx","fullName":"Kaltura Administrator","email":"dxxx","status":"1","password":"xxx!","isAdmin":"1","roleIds":"{1:result:id}","allowedPartnerIds":"*"},"ks":"ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow=="},"kalsig":"807e69d550f9f0e6060601b792f7c477"}
server: [xxx-srv-kaltura02], session: [591054419]
result (serialized): a:2:{i:0;a:4:{s:4:"code";s:10:"INVALID_KS";s:7:"message";s:129:"Invalid KS "ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow==", Error "-1,INVALID_STR"";s:10:"objectType";s:19:"KalturaAPIException";s:4:"args";a:3:{s:4:"KSID";s:92:"ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow==";s:8:"ERR_CODE";i:-1;s:8:"ERR_DESC";s:11:"INVALID_STR";}}i:1;a:4:{s:4:"code";s:10:"INVALID_KS";s:7:"message";s:129:"Invalid KS "ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow==", Error "-1,INVALID_STR"";s:10:"objectType";s:19:"KalturaAPIException";s:4:"args";a:3:{s:4:"KSID";s:92:"ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow==";s:8:"ERR_CODE";i:-1;s:8:"ERR_DESC";s:11:"INVALID_STR";}}}
result (object dump): Array
(
    [0] => Array
        (
            [code] => INVALID_KS
            [message] => Invalid KS "ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow==", Error "-1,INVALID_STR"
            [objectType] => KalturaAPIException
            [args] => Array
                (
                    [KSID] => ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow==
                    [ERR_CODE] => -1
                    [ERR_DESC] => INVALID_STR
                )

        )

    [1] => Array
        (
            [code] => INVALID_KS
            [message] => Invalid KS "ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow==", Error "-1,INVALID_STR"
            [objectType] => KalturaAPIException
            [args] => Array
                (
                    [KSID] => ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow==
                    [ERR_CODE] => -1
                    [ERR_DESC] => INVALID_STR
                )

        )

)

execution time for [http://xxx-srv-kaltura02.iptv.xxx.hr/api_v3/service/multirequest]: [0.021297931671143]
Executing failed for request #1 with error [Invalid KS "ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow==", Error "-1,INVALID_STR"]
PHP Fatal error:  Uncaught exception 'KalturaException' with message 'Invalid KS "ODZhNDhkOTI2ODAzNDRkYmI5Y2Y1NDkwYTNiMTBiNGE1NDE1OGJlZnwtMjstMjsxNDU3NzkyNjg4OzI7MTUzMjI7Ow==", Error "-1,INVALID_STR"' in /opt/kaltura/app/tests/standAloneClient/exec.php:351
Stack trace:
#0 {main}
  thrown in /opt/kaltura/app/tests/standAloneClient/exec.php on line 351

**Comment by damir-oiv** [comment]
api calls are failing, but not due to server unresolvable

**Comment by KalturaCommunity** [comment]
You need to understand why they're failing.
Like I said before though, please post relevant info from the log in 
forum.kaltura.org and I'll be happy to help you understand why.

May the source be with you,

Jess Portnoy

Kaltura Connect Virtual Summits 2015 - now on VOD! [connect.mediaspace.kaltura.com]
